### PR TITLE
[WIP] Initial import of extraction from SecurityBundle

### PR DESCRIPTION
**Pull request by linaori**
This PR is part of the initial import of what I've extracted from the SecurityBundle. I've made a todo list here. Who ever feels like helping out, please [fork my branch (feature/extract-from-core)](https://github.com/iltar/AclBundle/tree/feature/extract-from-core) and make Pull Requests so they show up in this merge.

TODO List
- [x] Add `README.md`
- [x] Add `.travis.yml`
- [x] Add XSD
- [x] Update changelog
- [ ] Update docs
- [x] Configure Travis
- [x] Add the license file: moved from `Resources/meta/LICENSE` to root
- [x] Minimize required dependencies in `composer.json`
- [ ] Add a BC layer for the ACL configuration under `security`

_If I'm missing anything, please let me know._

### CODE REVIEW

**Comment by fabpot** [comment]
I've added this repo on Travis.

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Security Bundle - ACL (Access Control List)
```
Comment (fabpot) [inline readme.md]: To be consistent, it should be named `README.md`

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
+CHANGELOG
```
Comment (fabpot) [inline CHANGELOG.md:1]: I would just start a new CHANGELOG from scratch.

**Comment by linaori** [comment]
@nicolas-grekas can you verify the `.travis.yml` and see if anything is missing?

**Comment by fabpot** [comment]
@iltar The LICENSE file should be put in the root directory (that's the new best practice now).

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+cache:
+  directories:
+    - "$HOME/.composer/cache"
+
+env:
+  - COMPOSER_FLAGS="--prefer-stable"
+
+php:
+  - 5.6
+  - hhvm
```
Comment (fabpot) [inline .travis.yml]: Can you remove the `php` section and add those version directly into the matrix?

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+matrix:
+  include:
+    - php: 5.5
+      env: COMPOSER_FLAGS="--prefer-lowest"
+    - php: 7.0
+      env: COMPOSER_FLAGS="" SYMFONY_VERSION="3.0.*"
+
+before_install: if [[ "$SYMFONY_VERSION" != "" ]]; then composer require --no-update symfony/symfony:${SYMFONY_VERSION}; fi
+
+install: composer update $COMPOSER_FLAGS --prefer-dist
```
Comment (fabpot) [inline .travis.yml]: `--prefer-dist` could be removed

**Comment by linaori** [comment]
@fabpot not sure if the matrix is the way you imagined it now; `build 4.1 No language set`

**Pull request closed without merging**